docs/readthedocs/source/doc/LLM/Overview/KeyFeatures/optimize_model.md

```diff
@@ -18,10 +20,46 @@ from bigdl.llm import optimize_model
 model = optimize_model(model)
 ```
 
+```eval_rst
+.. note::
+
+   In the above usage, symmetric INT4 optimizations are applied by default. You may apply other low bit optimizations (INT5, INT8, etc) through specifying ``low_bit`` as follows:
+
+   .. code-block:: python
+
+      # Apply symmetric INT8 optimization
+      model = optimize_model(model, low_bit="sym_int8")
+```
+
 After optimizing the model, BigDL-LLM does not require any change in the inference code. You can use any libraries to run the optimized model with very low latency.
 
+### Save & Load Optimized Model
+
+The loading process of the original model may be time-consuming and memory-intensive. For example, the [Llama-2-7b-chat-hf](https://huggingface.co/meta-llama/Llama-2-7b-chat-hf) model is stored with float16 precision, resulting in large memory usage when loaded using `LlamaForCausalLM`. To avoid high resource consumption and expedite loading process, you can use `save_low_bit` to store the model after low-bit optimization. Then, in subsequent uses, you can opt to use the `load_low_bit` API to directly load the optimized model. Besides, saving and loading operations are platform-independent, regardless of their operating systems.
+#### Save
+
+Continuing with the example of [Llama-2-7b-chat-hf](https://huggingface.co/meta-llama/Llama-2-7b-chat-hf), we can save the previously optimized model as follows:
```

Comment:
May be here it would be better to jump to the # Optimize Model section instead of jumping to the model link

Reply:
> May be here it would be better to jump to the # Optimize Model section instead of jumping to the model link

yeah, I've updated the link